### ISSUE TITLE
Contributing docs reference a branch that doesn't exist

### ISSUE THREAD

**Issue by korikuzma** [ga4gh/vrs#606]
https://github.com/ga4gh/vrs/blob/0106b48c1db0b420efd787cc769e4b35f415068e/CONTRIBUTING.md?plain=1#L29

I'm assuming we'd want to change this to `2.x`?

**Comment by srinitha709** [comment]
Yes,  it seems reasonable to assume that the branch reference should be updated from main to 2.x all new feature pull requests should target it.
Updating the documentation ensures contributors are targeting the correct branch, reducing confusion and errors.
